src/dfp/fill-static-advert-slots.ts

```diff
@@ -33,6 +36,18 @@ const decideAdditionalSizes = (adSlot: HTMLElement): SizeMapping => {
 			phablet: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
 			desktop: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
 		};
+	} else if (name === 'article-end' && isInUsa()) {
+		return {
+			mobile: [adSizes.fluid],
+		};
+	} else if (
+		name === 'article-end' &&
+		isInVariantSynchronous(mpuWhenNoEpic, 'variant') &&
+		isInUk()
+	) {
+		return {
+			desktop: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
+		};
```

Comment:
```suggestion
	} 
	 
	 if (name === 'article-end' && isInUsa()) {
		return {
			mobile: [adSizes.fluid],
		};
	} 
	
	if (
		name === 'article-end' &&
		isInVariantSynchronous(mpuWhenNoEpic, 'variant') &&
		isInUk()
	) {
		return {
			desktop: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
		};
```

Reply:
Would that make a big difference in our code? I mean what will add to it?

Reply:
The functionality is the same either way. On latest `main` (https://github.com/guardian/commercial/blob/main/src/dfp/fill-static-advert-slots.ts) the `else`'s aren't used, so my suggestion would be to keep things consistent

Reply:
Got it! Sorry for the slow understanding 😅

Reply:
It was very confusing. We were looking at two different versions of the same file!

Reply:
Yes but I checked what we already have in main and resolved the conflicts based on that.